### PR TITLE
feat/post image delete

### PR DESCRIPTION
**Pull request by mikco-hipe**
### OVERVIEW

Added post image delete
Initial alert message

### NOTES

`php artisan migrate` for  added `deleted_at` column to post_media table

### SNIPPETS
post_image_del

### CODE REVIEW

File: laravel-app/app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -133,4 +134,16 @@ public function share(SharePostRequest $sharePostRequest, UserPost $post): Redir
...
+    /**
+     * Delete image on post
+     */
+    public function deleteImage(UserPost $post): JsonResponse
+    {
+        if ($this->postService->deleteImage($post)) {
+            return response()->json(['message' => 'Image successfully deleted'], 200);
+        } else {
+            return response()->json(['message' => 'Image not found'], 404);
+        }
```
Comment (hipe01mark) [inline laravel-app/app/Http/Controllers/PostController.php:147]: Prevent using else in the if statement!

Comment (mikco-hipe) [reply laravel-app/app/Http/Controllers/PostController.php:147]: Yes sir, will resolve it now 🙇

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by nadj1233 — APPROVED** [review]
LGTM 🐐

**Pull request merged**